requirements.dev.txt

```diff
@@ -1,12 +1,12 @@
-jax>=0.3.14
-jaxlib>=0.3.14
+jax>=0.3.21
+jaxlib>=0.3.20
 flake8>=3.8.4
 pylint>=2.6.0
 pur>=5.3.0
 pytest>=6.1.2
 pytest-cov>=2.10.1
 pytest-xdist>=2.1.0
 twine>=3.2.0
-gym[box2d]>=0.21.0
+gym[box2d]>=0.26.0
```

Comment:
This doesn't match the version in `requirements.txt`.

In fact, do we need this line at all?

Reply:
If we do want to bump the gym dependency, we should probably do it in `requirements.txt` instead.

Reply:
(and delete it here)